core/protobuf/src/proto/dataserver/protocol.proto

```diff
@@ -58,3 +58,24 @@ message Response {
   optional status.PStatus status = 1;
   optional string message = 2;
 }
+
+// Netty RPCs. Every RPC needs to define a request type and optionally a response type (default to Response).
+
+// next available id: 2
+message LocalBlockOpenRequest {
+  optional int64 block_id = 1;
+  // Safe to remove before 1.5.0.
```

Comment:
yes, it will be removed in Calvin's change which will deprecate session_id. I removed this comment to avoid confusion.